### ISSUE TITLE
numpy hex API version constraints

### ISSUE THREAD

**Issue by lazzarello** [GUVWAF/Meshtasticator#11]
here's a weird one, numpy verions "15" versus "16" in an exception.

```
RuntimeError: module compiled against API version 0x10 but this version of numpy is 0xf
Traceback (most recent call last):
  File "/home/lee/src/Meshtasticator/interactiveSim.py", line 7, in <module>
    from lib.common import *
  File "/home/lee/src/Meshtasticator/lib/common.py", line 3, in <module>
    from . import phy 
    ^^^^^^^^^^^^^^^^^
  File "/home/lee/src/Meshtasticator/lib/phy.py", line 4, in <module>
    from scipy.optimize import fsolve
  File "/home/lee/src/Meshtasticator/.venv/lib/python3.11/site-packages/scipy/optimize/__init__.py", line 404, in <module>
    from ._optimize import *
  File "/home/lee/src/Meshtasticator/.venv/lib/python3.11/site-packages/scipy/optimize/_optimize.py", line 33, in <module>
    from scipy.sparse.linalg import LinearOperator
  File "/home/lee/src/Meshtasticator/.venv/lib/python3.11/site-packages/scipy/sparse/__init__.py", line 267, in <module>
    from ._csr import *
  File "/home/lee/src/Meshtasticator/.venv/lib/python3.11/site-packages/scipy/sparse/_csr.py", line 10, in <module>
    from ._sparsetools import (csr_tocsc, csr_tobsr, csr_count_blocks,
ImportError: numpy.core.multiarray failed to import
```

python version 3.11.3

numpy from requirements.txt

Ubuntu 22.04

**Comment by lazzarello** [comment]
just need to version bump numpy to 1.23.0. I could pull request if it helps/

**Comment by GUVWAF** [comment]
That would be great!

**Comment by lazzarello** [comment]
#12 there u go

**Comment by GUVWAF** [comment]
Fixed by #12.
